openvdb/openvdb/math/Maps.h

```diff
@@ -294,14 +294,8 @@ class OPENVDB_API MapRegistry
 
 
 /// @note Macro to use a final specifier from ABI=8 onwards.
-#if OPENVDB_ABI_VERSION_NUMBER >= 8
 #define OPENVDB_MAP_CLASS_SPECIFIER final
```

Comment:
I would remove the macro entirely and simply replace each instance with `final` now.